ubuntu-desktop-gnome/azuredeploy.parameters.json

```diff
@@ -1,19 +1,5 @@
 {
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
-    "parameters": {
-
-        "newStorageAccountName": {
-            "value": "GEN-UNIQUE"
-        },
-        "adminUsername": {
-            "value": "GEN-UNIQUE-8"
-        },
-        "adminPassword": {
-            "value": "GEN-PASSWORD"
-        },
-        "dnsNameForPublicIP": {
-            "value": "GEN-UNIQUE"
-        }
-    }
+    "parameters": { }
```

Comment:
revert this change

Reply:
reverted changes, CI test passed. Thanks!